### ISSUE TITLE
PyART fails to dealias certain radar files in hodo_plots.py

### ISSUE THREAD

**Issue by lcarlaw** [tjturnage/cloud-radar-server#48]
Certain radar files result in an exception thrown by PyART due to inconsistent nyquist intervals within the radar file (see output screenshot below).  A reproducible set of radar files is available [here](https://drive.google.com/drive/folders/1zaN7iNQ3CvgE4fSQBzR3Ypq_wc-msliW).  In this case, the radar was operating in VCP 12, so it's possible the PRFs were sectorized (as this is the only VCP for which different PRFs can be specified).  However, looking at the radar data in GR, it's not immediately clear this is the case. 

For ease of use with the monitor function which outputs script status, it'd be best if the hodo plot script would produce a blank image, perhaps with text warning the user there were issues dealiasing the data.  Honestly, this seems to be more of an issue with how PyART handles radar files.  

![error](https://github.com/user-attachments/assets/1927abc4-214f-4630-a464-7b734c2d6119)

**Comment by lcarlaw** [comment]
A possible solution/workaround to this issue is shown in [this commit](https://github.com/tjturnage/cloud-radar-server/commit/6116f986b11615f8a79a9e8458552dcb5c71d57e#diff-afb3dc9bd4d7a65f5dd74b845d8d6f9fb801fe87d23ed11810a91b782713ccee).  

It allows plots to continue to generate (which is good from a script-monitoring perspective), but adds warning text to the top of the hodograph plots that there were issues dealiasing the radar data.  Would it be better to even state that NO dealiasing was performed instead?

**Comment by scottthomaswx** [comment]
Lee, this is an issue I've never run into as most offices I've seen use VCP212 which would be consistent on Nyquist Velocities. I di suspect you're right. The version of the code designed for TDWRs (ID starts with T) removes de-aliasing which would fix that issue. I think what you have in the commit is great. Switching the wording to no dealiasing would be more accurate but that's small potatoes. Is there anything you need me to look at? Otherwise what you have looks good.

**Comment by lcarlaw** [comment]
Thanks Scott! I realize I didn't share the radar files, but did so in the event you want to take a look at them. We do utilize VCP12 here for its faster volume scans, and I know southern region offices do so as well. What's odd is it doesn't actually look like the PRFs were sectorized in those radar files based on where the 1st trip RF area was. I did not take a look at the actual output via pyart though. 

Either way, this patch should cover us and I've made the wording alteration you mentioned above. Ultimately, seems like an issue with the post-processing step via Pyart itself. Thank!
